### ISSUE TITLE
[Bug]: Upgrading to 0.9.0 results in TypeError: styleDictionary.cleanAllPlatforms is not a function

### ISSUE THREAD

**Issue by rejas** [tokens-studio/sd-transforms#185]
### What happened?

Bumped the version used in our project to the latest version and noticed that with v0.9.0 this error appears:

```
> style-dictionary clean --config ./build.js

/Users/michael/Projects/design-tokens/node_modules/browser-style-dictionary/bin/style-dictionary:107
    styleDictionary.cleanAllPlatforms();
                    ^

TypeError: styleDictionary.cleanAllPlatforms is not a function
    at Command.styleDictionaryClean (/Users/michael/Projects/design-tokens/node_modules/browser-style-dictionary/bin/style-dictionary:107:21)
    at Command.listener [as _actionHandler] (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:488:17)
    at /Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:1227:65
    at Command._chainOrCall (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:1144:12)
    at Command._parseCommand (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:1227:27)
    at Command._dispatchSubcommand (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:1050:25)
    at Command._parseCommand (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:1193:19)
    at Command.parse (/Users/michael/Projects/design-tokens/node_modules/commander/lib/command.js:897:10)
    at Object.<anonymous> (/Users/michael/Projects/design-tokens/node_modules/browser-style-dictionary/bin/style-dictionary:111:9)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)

```

### Version

0.9.0

**Comment by rejas** [comment]
still happening with latest v0.11.6 maybe related/duplicate of https://github.com/tokens-studio/sd-transforms/issues/197 ?

**Comment by jorenbroekema** [comment]
Is your Style-Dictionary config also a CommonJS file `module.exports = {....}` ?

**Comment by jorenbroekema** [comment]
fixed in v0.11.8. Might need to clean out your node_modules folder or package-lock file to ensure browser-style-dictionary is no longer installed as a dependency of sd-transforms.

**Comment by rejas** [comment]
Thx, latest version works  (indeed only after deleting node_modules once) :-)
